egs2/makerere/asr1/README.md

```diff
@@ -0,0 +1,47 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Thu Mar 28 00:34:47 EDT 2024`
+- python version: `3.10.13 | packaged by conda-forge | (main, Dec 23 2023, 15:36:39) [GCC 12.3.0]`
+- espnet version: `espnet 202402`
+- pytorch version: `pytorch 2.0.1`
+- Git hash: `eed7751c910977290ef9a177ea0942a0e3c2fd35`
+  - Commit date: `Mon Mar 25 18:26:50 2024 +0000`
```

Comment:
Please upload the model to the HF hub and add a link here

Reply:
I have uploaded the model to the HF hub. Here's the link: https://huggingface.co/satvik-dixit/asr_makerere.